.circleci/config.yml

```diff
@@ -299,7 +299,7 @@ workflows:
             - build_and_test
           filters:
             branches:
-              only: develop
+              only: landing-page-tweaks
```

Comment:
Revert before merging

Reply:
Absolutely. I'm keeping this for a bit so design can see the latest tomorrow morning.